vm/interpreter/src/vm.rs

```diff
@@ -181,6 +181,13 @@ where
             if i > parent_epoch {
                 self.run_cron(epoch, callback.as_mut())?;
             }
+            if is_migrate_epoch(i) {
```

Comment:
I'm confused about why the need to check if migration through the networks, can't you just match the epochs inside of `migrate_state` and in that function just return an `Option<Cid>` as the return type, and overwrite on `Some`? Seems cleaner than checking through the table, then matching again in the function (when implemented) to avoid returning an option.

Reply:
If we have a function that does the check and migrate then we would have to flush after the cron regardless of whether we need to migrate or not. I thought we agreed that was wasteful?

Reply:
No you wouldn't because you would still only flush when matched, I'm just suggesting this to be able to keep that original function signature, but that doesn't matter much I don't think